portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -66,8 +70,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String text = request.getParameter("form-comment"); 
-    String name = request.getParameter("form-name"); 
+    String body = "";
+    try (BufferedReader reader = request.getReader()) {
+      if (reader == null) {
+            return;
+      }
+      String line;
+      while ((line = reader.readLine()) != null) {
+        body = line; 
```

Comment:
yea it's actually weird because that line is being called once which makes me feel like even more that there's a better way to do this

Reply:
By default the BufferedReader reads either characters or lines, and in your case there's only one line. It would still be best to not assume anything about the number of lines and concatenate them together. 

If you are feeling ambitious about using a one-liner to do this, check out `BufferedReader.lines()` method, which returns a [`Stream`](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html). Streams have a `collect` method that takes a [`Collector`](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Collector.html). The [`Collectors` class](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Collectors.html) has useful class methods for creating a `Collector`.